apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -42,7 +42,7 @@ export const calendarizeCourseEvents = (currentCourses: ScheduleCourse[] = []) =
                             term: course.term,
                             title: course.deptCode + ' ' + course.courseNumber,
                             courseTitle: course.courseTitle,
-                            bldg: meeting.bldg,
+                            bldg: meeting.bldg[0],
```

Comment:
Opened an issue for the fact that meeting.bldg is an array now: #592. Don't want to scope-creep this PR.